### PR TITLE
Feature random quotes

### PR DESCRIPTION
**Pull request by GaayuoniC**
1. Data fetching from a free API
2. Display of random quote on home page
3. Simplicity in styling

### CODE REVIEW

File: src/Pages/HomePage.jsx [hunk 1]
```diff
@@ -1,43 +1,50 @@
...
+            <span>Привет! </span>
+            <span>Amaaraba! </span>
+            <span>Akwaaba! </span>
+          </h1>
+          <p>
+            Welcome! My name is Christian and am originally from Ghana. Having
+            spent the first half of my life pursuing two of my dreams in life, I
+            have now gotten to the point where I finally get the opportunity to
+            achieve my last dream, that is finally venturing into the world of
+            IT. So here I am, starting a carrier change and venturing into the
```
Comment (actyralf) [inline src/Pages/HomePage.jsx]: carrier -> career

File: src/Pages/HomePage.jsx [hunk 2]
```diff
@@ -1,43 +1,50 @@
...
+        <div className="home-page-contianer-item-1">
+          <h1 className="continuous-slide">
+            <span>Hello! </span>
+            <span>Hallo! </span>
+            <span>Привет! </span>
+            <span>Amaaraba! </span>
+            <span>Akwaaba! </span>
+          </h1>
+          <p>
+            Welcome! My name is Christian and am originally from Ghana. Having
```
Comment (actyralf) [inline src/Pages/HomePage.jsx]: ...and I am originally...

File: src/Pages/HomePage.jsx [hunk 3]
```diff
@@ -1,43 +1,50 @@
...
+          </p>
+          <br />
+          <p>
+            This is just the beginning of the development of my own website
+            which I started developing right from the first day that I started
+            my
+            <strong> Frontend Web and App </strong> developement Bootcamp with{" "}
+            <strong>
+              <a href="https://talents.taktsoft.com/" target="_blank">
+                Taktsoft Campust Talents {""}
```
Comment (actyralf) [inline src/Pages/HomePage.jsx]: Campust -> Campus

File: src/components/RandomQuotes.jsx [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+      } finally {
+        setIsLoading(false);
+      }
+    }
+    getData(); //call the function to immediately fetch the data needed
+
+    //setting the interval time for changing the quotes
+    const intervalId = setInterval(() => {
+      getData();
+    }, 3600000); // 3600 00ms = 1 hour
```
Comment (actyralf) [inline src/components/RandomQuotes.jsx]: Quite a long interval - I would use 30 seconds or so...

**Review by actyralf — COMMENTED** [review]
Nice! Nothing except some typos.
You should start targeting mobile screens better though - it looks a little bit broken on mobile.

**Comment by GaayuoniC** [comment]
Hello Ralf,

Thanks for the feedback. I will work on the mobile aspect now.

Regards,
Chris

Sent with [Proton Mail](https://proton.me/) secure email.

**Pull request merged**